src/App.js

```diff
@@ -1,26 +1,22 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Switch, Route } from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Checkout from './pages/Checkout';
+import ProductDetails from './pages/ProductDetails';
+import ProductList from './pages/ProductList.jsx';
```

Comment:
eslint reclamando da extensão .jsx , acho q não precisa especifica, mais testa antes pra não quebrar nada.
```suggestion
import ProductList from './pages/ProductList';
```